import.php

```diff
@@ -589,7 +589,7 @@ protected function show_setup_form()
         ?>
 <script type="text/javascript">
 
-g_presets = <?php echo json_encode($this->presets) . ';'?>
+g_presets = <?php echo json_encode($this->presets, JSON_UNESCAPED_UNICODE) . ';'?>
```

Comment:
so we can compare against 'Säästöpankki' instead of 'S\u00e4\u00e4st\u00f6pankki'